pallets/staking/src/lib.rs

```diff
@@ -230,10 +227,11 @@ where
 	type AccountId = AccountId;
 	type BlockNumber = BlockNumber;
 	type Error = &'static str;
+	type MaxWinners = (); //TODO
 	type DataProvider = DataProvider;
 
-	fn ongoing() -> bool {
-		false
+	fn desired_targets_checked() -> frame_election_provider_support::data_provider::Result<u32> {
```

Comment:
@CAGS295 Need an implementation for this

Reply:
I remember addressing this, I forgot how though. I will double check.